annotator/static/views/rect.js

```diff
@@ -531,6 +537,8 @@ class CreationRect extends Rect {
     }
 
     onDragEnd() {
+        var a = Date.now()
```

Comment:
use let here; var will make it global scoped and might cause some issues

Reply:
Ok. Shall change.